styles/styles.css

```diff
@@ -240,20 +241,35 @@
     margin: 20px 0;
   }
 
-
   .breadcrumbs {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
     font-size: .8em;
     color: #c8c8c8;
     padding: 8px 0;
     user-select: none;
+  }
+
+  .breadcrumbs li {
```

Comment:
why not just nest this li inside .breadcrumbs - as to not duplicate the selector